<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TkkrLab Barsystem</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;outline: 0;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="windowOrder">
     <widget class="QPushButton" name="btnCancelOrder">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>520</y>
        <width>140</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: red</string>
      </property>
      <property name="text">
       <string>Cancel
order</string>
      </property>
     </widget>
     <widget class="QWidget" name="productFrame" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>520</width>
        <height>600</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPlaceOrder">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>520</y>
        <width>140</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#00ff00
</string>
      </property>
      <property name="text">
       <string>Place
order</string>
      </property>
     </widget>
     <widget class="QListView" name="lstProductQueue">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>0</y>
        <width>280</width>
        <height>520</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
     <zorder>productFrame</zorder>
     <zorder>btnCancelOrder</zorder>
     <zorder>btnPlaceOrder</zorder>
     <zorder>lstProductQueue</zorder>
    </widget>
    <widget class="QWidget" name="windowWelcome">
     <widget class="QPushButton" name="btnCashOrder">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>500</y>
        <width>160</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cash order</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblVersionCode">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Version: unknown</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblImage">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>500</width>
        <height>401</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblText1">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>361</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TkkrLab</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblText2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>510</y>
        <width>351</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Barsystem</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMinimize">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Minimize</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
